docs/assets/zdm-config-reference.yml

```diff
@@ -0,0 +1,170 @@
+# This variable determines which cluster is currently considered the primary cluster.
+# At the start of the migration, the primary cluster is Origin, as it contains all the data.
+# In Phase 4 of the migration, once all the existing data has been transferred and any validation/reconciliation
+# step has been successfully executed, you can switch the primary cluster to be Target.
+# Valid values: ORIGIN, TARGET.
+primary_cluster: ORIGIN
+
+# This variable determines how reads are handled by the ZDM Proxy. Valid values:
+# PRIMARY_ONLY - reads are only sent synchronously to the primary cluster. This is the default behavior.
+# DUAL_ASYNC_ON_SECONDARY - reads are sent synchronously to the primary cluster and also asynchronously
+# to the secondary cluster. See Phase 3: Enable asynchronous dual reads.
+read_mode: PRIMARY_ONLY
+
+# Whether the ZDM Proxy should replace standard CQL function calls in write
+# requests with a value computed at proxy level. Currently, only the replacement
+# of now() is supported. Disabled by default. Enabling this will have a noticeable performance impact.
+# replace_cql_functions: false
+
+# Timeout (in ms) when performing the initialization (handshake) of a proxy-to-secondary cluster
+# connection that will be used solely for asynchronous dual reads. If this timeout occurs, the asynchronous
+# reads will not be sent. This has no impact on the handling of synchronous requests: the ZDM Proxy will
+# continue to handle all synchronous reads and writes normally.
+# async_handshake_timeout_ms: 4000
+
+# Specifies logging level.
+# log_level: INFO
+
+# List of peer ZDM proxy instances. This configuration parameter should be identical through all ZDM proxies.
+# proxy_topology_addresses: 127.0.1.1, 127.0.1.2, 127.0.1.3
+
+# Index of local ZDM proxy instance within "proxy_topology_addresses" list.
```

Comment:
Maybe a small example like the one I gave you in slack would be good to illustrate this. `0` on the `127.0.1.1`, `1` on the `127.0.1.2` and `2` on the `127.0.1.3`